pyaerocom/aeroval/setup/units_setup.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+
+class UnitsSetup(BaseModel):
+    units: dict[str, str] = {}
```

Comment:
Some comment about how to this should be specified would be helpful. Especially an example in a comment which will appear on RTD